tests/Feature/Production/RemnantFilterTest.php

```
@@ -0,0 +1,380 @@
+    $this->admin->assignRole('admin');
+
+    $this->produccion = User::factory()->create(['email_verified_at' => now()]);
+    $this->produccion->assignRole('produccion');
+
+    $this->operador = User::factory()->create(['email_verified_at' => now()]);
+    $this->operador->assignRole('operador');
+
+    $this->comercial = User::factory()->create(['email_verified_at' => now()]);
+    $this->comercial->assignRole('comercial');
+
+    $category = ProductCategory::create(['name' => 'Categoría Pinturas']);
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\ProductCategory::create().


(staticMethod.notFound)